### PR TITLE
Update ja.coffee

### PR DESCRIPTION
**Pull request by Coderaulic**
Finished translations for "Contact Us".
Assuming no translation is wanted for deprecated lines.
Added one off translations here and there.
Not sure how to translate line 340. Any insight into meaning is appreciated.

### CODE REVIEW

File: app/locale/ja.coffee [hunk 1]
```diff
@@ -336,7 +336,7 @@ module.exports = nativeDescription: "日本語", englishDescription: "Japanese",
     tip_google: "解決できない問題がありますか？ググっちゃえ！"
     tip_adding_evil: "悪を少々加える。"
     tip_hate_computers: "コンピュータを憎むと思う人についての事だ。彼らが本当に嫌いなのは下手なプログラマーだ。- ラリー・ニーヴン"
-#    tip_open_source_contribute: "You can help CodeCombat improve!"
+    tip_open_source_contribute: "あなたはCodeCombatを向上させることができます！"
 #    tip_recurse: "To iterate is human, to recurse divine. - L. Peter Deutsch"
```
Comment (Imperadeiro98) [inline app/locale/ja.coffee]: It is something like: "Iterating is human, recursing is divine.". It has programming concepts involved.

**Comment by Imperadeiro98** [comment]
Thank you, @Coderaulic!

**Comment by Coderaulic** [comment]
So I get iterative as in counting through a stack of money, as in that is the natural human way of doing things, we iterate...
Recursive I know as in finding a GCD or a Fibonacci number, and that is "Divine" as it can't be done relatively easy without programming.
Is that the idea  Peter Deutsch is trying to convey in this quote?

**Comment by Imperadeiro98** [comment]
I think so. But it is my opinion :)

**Pull request merged**